src/card/styled-components.js

```diff
@@ -4,6 +4,15 @@ import {styled} from '../styles/index';
 
 export const Action = styled('div', () => ({}));
 
+export const ActionLink = styled('a', ({$theme}) => {
```

Comment:
How about we create a separate `Link` component for it and use it in the Card example? And meanwhile just create a styled link element for an example only purpose instead of a componet's export. Since later getting rid of the link from a card would be a breaking change and people may start using this styled link outside of the card component.